packages/request-manager/e2e/torRunner.ts

```diff
@@ -0,0 +1,48 @@
+import path from 'path';
+import { spawn } from 'child_process';
+
+const processDir = path.join(__dirname, './../../suite-data/files/bin/tor/linux-x64/');
```

Comment:
Note: make `linux-x64` default  but let's allow a flag to the command or similar to allow other systems binaries.

Reply:
Done by allowing env variable `TOR_BINARY_PATH` .
Usage explained in `packages/request-manager/README.md`